res/values/strings.xml

```diff
@@ -120,14 +120,14 @@
     <string name="voice_message">Voice message</string>
     <string name="forwarded">Forwarded</string>
     <string name="forwarded_message">Forwarded message</string>
-    <!-- %1$s will be replaced by the name or the email-address of the person who has forwarded the message -->
+    <!-- %1$s will be replaced by the name or the e-mail address of the person who has forwarded the message -->
     <string name="forwarded_by">Forwarded by %1$s</string>
     <string name="video">Video</string>
     <string name="documents">Documents</string>
     <string name="contact">Contact</string>
     <string name="verified_contact">Verified contact</string>
     <string name="camera">Camera</string>
-    <!-- "capture" here means "start a video recording" or "take a photo"; eg. the description of the "shutter button" in the camera controls -->
+    <!-- "capture" here means "start a video recording" or "take a photo"; e.g. the description of the "shutter button" in the camera controls -->
```

Comment:
not sure which one is better, there are lots of discussions around it :) iirc, in core we decided for "eg." (i personally also did use "e.g." before, but i do not care at the end :)

Reply:
The abbreviation is "exempli gratia", and "e.g." is correct.
In those discussions, like DeltaChat core, people saying "eg." are wrong.
Glad we cleared that up :)